rs/types/types/src/consensus/idkg/common.rs

```diff
@@ -1145,32 +1139,38 @@ impl ThresholdSigInputsRef {
             ThresholdSigInputsRef::Schnorr(inputs_ref) => inputs_ref
                 .translate(resolver)
                 .map_or_else(err_schnorr, ok_schnorr),
+            ThresholdSigInputsRef::VetKd(inputs) => Ok(ThresholdSigInputs::VetKd(inputs.clone())),
         }
     }
 }
 
+#[allow(clippy::large_enum_variant)]
 pub enum ThresholdSigInputs {
     Ecdsa(ThresholdEcdsaSigInputs),
     Schnorr(ThresholdSchnorrSigInputs),
+    VetKd(VetKdArgs),
```

Comment:
Maybe we should `Box`ing this?

Reply:
The actual error I'm getting is:
```
1148 | / pub enum ThresholdSigInputs {
1149 | |     Ecdsa(ThresholdEcdsaSigInputs),
     | |     ------------------------------ the second-largest variant contains at least 0 bytes
1150 | |     Schnorr(ThresholdSchnorrSigInputs),
1151 | |     VetKd(VetKdArgs),
     | |     ---------------- the largest variant contains at least 208 bytes
1152 | | }
     | |_^ the entire enum is at least 0 bytes
     |
```

So it seems that the compiler cannot figure out the correct minimum size for the ECDSA/Schnorr inputs. In reality, those struct are actually _larger_ than `VetKdArgs` (as expected):

```rust
println!("Size of VetKdArgs: {}", mem::size_of::<VetKdArgs>());
println!("Size of Ecdsa inputs: {}", mem::size_of::<ThresholdEcdsaSigInputs>());
println!("Size of ThresholdSigInputs: {}", mem::size_of::<ThresholdSigInputs>());
```

Output:
```
Size of VetKdArgs: 208
Size of Ecdsa inputs: 1352
Size of ThresholdSigInputs: 1352
```